src/utils/swagger.ts

```diff
@@ -12,7 +12,7 @@ const options: swaggerJsdoc.Options = {
       version,
     },
     components: {
-      securitySchemas: {
+      securitySchemes: {
```

Comment:
I wouldn't consider this a typo (might be wrong), after reading the [difference](https://wikidiff.com/scheme/schema) in my opinion schema is the correct term. 

Once again, might be wrong.

Reply:
Yes, it is correct term but not compatible to Swagger UI.
The correct term should be "securitySchemes" if you want to use Swagger UI with auth.
https://swagger.io/docs/specification/authentication/

Reply:
Hi please update securitySchemas to securitySchemes.